core/src/main/java/de/danoeh/antennapod/core/service/playback/LocalPSMP.java

```diff
@@ -260,22 +262,16 @@ private void playMediaObject(@NonNull final Playable playable, final boolean for
             callback.onMediaChanged(false);
             setPlaybackParams(PlaybackSpeedUtils.getCurrentPlaybackSpeed(media), UserPreferences.isSkipSilence());
             if (stream) {
-                if (playable instanceof FeedMedia) {
-                    FeedMedia feedMedia = (FeedMedia) playable;
-                    FeedPreferences preferences = feedMedia.getItem().getFeed().getPreferences();
-                    mediaPlayer.setDataSource(
-                            media.getStreamUrl(),
-                            preferences.getUsername(),
-                            preferences.getPassword());
-                } else {
-                    mediaPlayer.setDataSource(media.getStreamUrl());
-                }
```

Comment:
Looks like your PR accidentally reverts the fix for streaming password protected media.